assets/js/components/LiveCommunity.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+	<div class="d-block d-lg-flex mb-2 justify-content-between">
+		<SavingsTile
+			class="text-accent2"
+			icon="car"
+			:title="$t('footer.community.power')"
+			:value="chargePower"
+			:valueFmt="numberAnimationFmt"
+			:animationDuration="animationDuration"
+			unit="kW"
+			:sub1="$t('footer.community.powerSub1', { totalClients, activeClients })"
+			:sub2="$t('footer.community.powerSub2')"
+		/>
+
+		<SavingsTile
+			class="text-accent1"
+			icon="sun"
+			:title="$t('footer.community.greenShare')"
+			:value="greenShare"
+			:valueFmt="numberAnimationFmt"
+			:animationDuration="animationDuration"
+			unit="%"
+			:sub1="$t('footer.community.greenShareSub1')"
+			:sub2="$t('footer.community.greenShareSub2')"
+		/>
+
+		<SavingsTile
+			class="text-accent3"
+			icon="eco"
+			:title="$t('footer.community.greenEnergy')"
+			:value="greenEnergyMWh"
+			:valueFmt="numberAnimationFmt"
+			:animationDuration="animationDuration"
+			unit="MWh"
+			:sub1="$t('footer.community.greenEnergySub1')"
+			:sub2="$t('footer.community.greenEnergySub2')"
+		/>
+	</div>
+</template>
+
+<script>
+import SavingsTile from "./SavingsTile.vue";
+
+import formatter from "../mixins/formatter";
+import communityApi from "../communityApi";
+
+export default {
+	name: "LiveCommunity",
```

Comment:
How about `name: "Community data"` , for consistency ?

Reply:
Is see your point for symmetry, however I'd like to highlight the fact that its live-data in contrast to the more historic information in the "my data" tab. And "live community data" would blow up on mobile screens :D